lib/features/composer/presentation/composer_view.dart

```diff
@@ -210,7 +210,7 @@ class ComposerView extends GetWidget<ComposerController> {
                               return const SizedBox.shrink();
                             }
                           }),
-                          const SizedBox(height: ComposerStyle.keyboardMaxHeight),
+                          SizedBox(height: MediaQuery.viewInsetsOf(context).bottom),
```

Comment:
Also remove keyboardMaxHeight in ComposerStyle if not used

Reply:
Done